oxenstored/store.ml

```diff
@@ -168,6 +168,8 @@ module Path = struct
 
   let to_string t = "/" ^ String.concat "/" t
 
+  let compare a b = String.compare (to_string a) (to_string b)
```

Comment:
Where is this used? concatenating the strings to compare them might result in more churn.

Reply:
This is a remnant of an old implementation with a Map, deleted it. It (or an equivalent) would be needed if we decide to do a `Hashtbl.Make`